clipboard: add instructions for installation in bare and managed projects (#24284)
expo/expo: packages/expo-clipboard/README.md

Please refer to the [API documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/clipboard/).

> ## Installation in managed Expo projects
> 
> For [managed](https://docs.expo.dev/archive/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/clipboard/).

Reviewer: ```suggestion
```

We'd rather not refer to the archived documentation between managed vs bare or refer to managed projects. Any react native project can be used with Expo SDK. If a project is generated by using React Native CLI, the Expo SDK can be installed.

Let's remove this since API documentation link is enough for this context.

> ## Installation in bare React Native projects
> 
> For bare React Native projects, you must ensure that you have [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
> 
> ### Add the package to your npm dependencies
> 
> ```
> npx expo install expo-clipboard
> ```
> 
> ### Configure for iOS
> 
> Run `npx pod-install` after installing the npm package.
> 
> ### Configure for Android
> 
> No additional set up necessary.
> 

Reviewer: ````suggestion
## Installation in bare React Native projects

For bare React Native projects, ensure you've [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/).

### Add the package to your npm dependencies

```
npx expo install expo-clipboard
```

### Configure for iOS

Run `npx pod-install` after installing the library.

### Configure for Android

No additional set up necessary.

````

For now, I think this is good. We've planned out that we'll be updating the API docs and include bare related information for each Expo library within the doc and rather link back to the docs section here.
